src/server/pkg/hashtree/interface.go

```diff
@@ -82,6 +82,7 @@ type OpenNode struct {
 // updated (until then, the hashes and sizes stored in an OpenHashTree will be
 // stale).
 type OpenHashTree interface {
+	HashTree
```

Comment:
As mentioned offline, I'm not sure if an `OpenHashTree` returns the right sizes for its internal nodes.  Can @msteffen confirm?  Regardless, the usage in this patch should work since we are not really using sizes.